.github/workflows/wpil.yaml

```diff
@@ -0,0 +1,33 @@
+name: Update Wiki Page Icons List
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  update:
```

Comment:
Probably we should consider allowing this job to fail: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast